ecs/sdk.go

```diff
@@ -69,7 +72,7 @@ type sdk struct {
 
 func newSDK(sess *session.Session) sdk {
 	sess.Handlers.Build.PushBack(func(r *request.Request) {
-		request.AddToUserAgent(r, "Docker CLI")
+		request.AddToUserAgent(r, userAgent+"/"+internal.Version)
```

Comment:
Does that break some metrics stuff ??? I've set to "docker-cli" just to be the same as it's ACI counterpart.

Reply:
On ECS, I don't know if it breaks things (@ndeloof ? ) 
Why change "Docker CLI" to "docker-cli" ? (I don't mind really, but trying to minimize changes, probably doesn't change anything to breaking some metrics)

Reply:
On ACI, we need to check with @macolso how this impacts Docker-ACI metrics, probably needs some adaptation.

Reply:
We told AWS we use `Docker CLI` in user-agent string.
Introducing version is fine, could help check users updated to latest release, but this base string should not chnage

Reply:
@ndeloof So maybe doing the other way around could be a good thing...
If we change to `Docker CLI` on both, maybe that would work. @gtardif Could you please check with @macolso if that's OK to change that to `Docker CLI` ?